test/connectors/ReviewConnectorSpec.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.BaseSpec
+import mocks.{MockAppConfig, MockHttpClient}
+import models.errors.{InternalServerError, MalformedResponseError, NotFoundError, StaleDataError}
+import models.{ApprovalProcessReview, RequestOutcome, ReviewData}
+import play.api.test.{DefaultAwaitTimeout, FutureAwaits}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class ReviewConnectorSpec extends BaseSpec {
+
+  private trait ReviewInfoTest extends MockHttpClient with FutureAwaits with DefaultAwaitTimeout with ReviewData {
+
+    implicit val hc: HeaderCarrier = HeaderCarrier()
+
+    val id: String = "Oct90005"
```

Comment:
I'd be tempted to define "id" in ReviewData. If you going to create a separate trait to define your test data you might as well define all of the test data there. Also you can use the id in the process review definition.

Reply:
done, will be in next commit shortly